docs/source/api.rst

```diff
@@ -1,32 +1,70 @@
 API Reference
-=============
+#############
 
 poliastro.twobody package
 -------------------------
 
 .. automodule:: poliastro.twobody
+    :members:
+
+poliastro.twobody.angles module
+********************************
```

Comment:
The `***` headers are a sublevel of `---`, but I think I should ask you these kind of changes in the chat before making a Pull Request, noob error. 😄
I have compiled the docs without errors using Sphinx, I think it only warns you when the line is too short. Anyway, I will change it, it's a good practice as you say.